PingProtect/PingProtect/Sources/PIProtect.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PIProtect.swift
+//  PingProtect
+//
+//  Copyright (c) 2024 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import FRAuth
+import PingOneSignals
+
+/*
+ PIProtect is for initializing and interacting with Ping Portect SDK
+ */
+public final class PIProtect: NSObject {
+    
+    
+    /// Inititialize Ping Protect SDK
+    /// - Parameters:
+    ///   - initParams: `PIInitParams` containing parameters for the init
+    ///   - callback: Result callback of the init
+    public static func initSDK(initParams: PIInitParams? = nil, callback: @escaping (Error?) -> Void) {
+        
+        let signalsInitParams = initParams != nil ? initParams!.getPOInitParams() : POInitParams()
```

Comment:
not sure we need this getPOInitParams . is this same?

let signalsInitParams = initParams ??  POInitParams()

Reply:
initParams is **PIInitParams** type (our wrapper for **POInitParams**) while the PingOneSignals.initSDK() expects POInitPatams type, so I have getPOInitParams() method to conventionally cast from our wrapper to what signals SDK expects